README.md

```diff
@@ -46,15 +46,22 @@ cargo test
 ```
 
 ### Run w/ Docker (wip):
+* Make a new account in Metamask (or whichever method you prefer). Copy/paste the address into `tests/docker/.env` as `ACCOUNT_ID`. 
```

Comment:
it might be nice to have a test key we use for this by default so we can reuse the same private key + address, and less setup for a user